src/Microsoft.Extensions.Diagnostics.HealthChecks/HealthChecksBuilderAddCheckExtensions.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+// This type is used like this:
```

Comment:
We do this everywhere, don't think you need these comments.

Reply:
Well, but we don't normally put these in `.DependencyInjection`, see https://github.com/aspnet/Logging/blob/dev/src/Microsoft.Extensions.Logging/LoggingBuilderExtensions.cs and ... well OK it turns out Authentication does it that way. OK, maybe Logging actually the outlier here :).